scss-test.scss

```diff
@@ -0,0 +1,8 @@
+.parent {
+  color: #FFFFFF;
```

Comment:
Color literals like `#FFFFFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFFFFF` should be written as `#ffffff`<br>Properties should be ordered background, color